src/elements.rs

```diff
@@ -157,6 +233,57 @@ impl Element {
         }
     }
 
+    /// Retrieve the tag name of this element.
+    ///
+    /// See [13.6 Get Element Tag Name](https://www.w3.org/TR/webdriver1/#get-element-tag-name)
+    /// of the WebDriver standard.
+    #[cfg_attr(docsrs, doc(alias = "Get Element Tag Name"))]
+    pub async fn tag_name(&mut self) -> Result<String, error::CmdError> {
+        let cmd = WebDriverCommand::GetElementTagName(self.element.clone());
+        match self.client.issue(cmd).await? {
+            Json::String(v) => Ok(v),
+            v => Err(error::CmdError::NotW3C(v)),
+        }
+    }
+
+    /// Gets the x, y, width, and height properties of the current element.
+    ///
+    /// See [13.7 Get Element Rect](https://www.w3.org/TR/webdriver1/#dfn-get-element-rect) of the
+    /// WebDriver standard.
+    #[cfg_attr(docsrs, doc(alias = "Get Element Rect"))]
+    pub async fn get_element_rect(&mut self) -> Result<(f64, f64, u64, u64), error::CmdError> {
```

Comment:
I don't think the spec says anything about `x` and `y` being fractional while `width` and `height` are whole numbers. They all appear to be "CSS Pixel" values, and I _think_ those can always be fractional, in which case all of these should be `f64`.

Reply:
Also, since this is already a method _on_ `Element`, I think it should be called `get_rect`, or better yet just `rect`. Or why not properly: `rectangle`? The point f the `doc(alias` is specifically so that we can name the methods something else if we disagree with the standard, and still have it be searchable by the standard name.

Reply:
As for the types, I noticed by experimentation that `x.as_u64()` returned `None`, because it was a float. The width and height seemed to always be integers.

Happy to make them all f64.

Reply:
Yeah, let's have them all be floats to cover our bases here given that the spec claims they all have the same unit :+1: